interfaces/Portalicious/src/app/components/page-layout/components/footer/footer.component.html

```diff
@@ -1,124 +1,11 @@
-      </h3>
-      <ul>
-        <li class="mb-2 [&:not(:hover)]:underline">
-          <a
-            href="https://121.global"
-            target="_blank"
-          >
-            <!-- eslint-disable-next-line @angular-eslint/template/i18n -->
-            121.global
-          </a>
-        </li>
-        <li class="mb-2 [&:not(:hover)]:underline">
-          <a
-            href="https://510.global"
-            target="_blank"
-          >
-            <!-- eslint-disable-next-line @angular-eslint/template/i18n -->
-            510.global
-          </a>
-        </li>
-        <li class="mb-2 [&:not(:hover)]:underline">
-          <a
-            href="https://data.ifrc.org"
-            target="_blank"
-          >
-            <!-- eslint-disable-next-line @angular-eslint/template/i18n -->
-            data.ifrc.org
-          </a>
-        </li>
-      </ul>
-    </div>
-    <div>
-      <h3
-        class="mb-4"
-        i18n
-      >
-        Helpful links
-      </h3>
-      <ul>
-        <li class="mb-2 [&:not(:hover)]:underline">
-          <a
-            i18n
-            href="https://github.com/global-121/121-platform/"
-            target="_blank"
-            >Open Source Code</a
-          >
-        </li>
-        <li class="mb-2 [&:not(:hover)]:underline">
-          <a
-            i18n
-            [href]="apiDocumentationUrl"
-            target="_blank"
-            >API Documentation</a
-          >
-        </li>
-        <li class="mb-2 [&:not(:hover)]:underline">
-          <a
-            i18n
-            href="https://manual.121.global/"
-            target="_blank"
-            >User Manual</a
-          >
-        </li>
-      </ul>
-    </div>
-    <div>
-      <h3
-        class="mb-4"
-        i18n
-      >
-        Questions or feedback? Contact us
-      </h3>
-      <a
-        href="mailto:support@510.global"
-        class="p-button p-button-rounded p-button-outlined p-button-secondary [&:not(:hover)]:text-white"
-      >
-        <!-- eslint-disable-next-line @angular-eslint/template/i18n -->
-        support&#64;510.global
-      </a>
-      <div class="mt-6 flex gap-3">
-        <a
-          href="https://www.linkedin.com/company/510global/"
-          target="_blank"
-        >
-          <i class="pi pi-linkedin text-2xl text-white"></i>
-        </a>
-        <a
-          href="https://www.instagram.com/510redcross/"
-          target="_blank"
-        >
-          <i class="pi pi-instagram text-2xl text-white"></i>
-        </a>
-        <a
-          href="https://x.com/RedCross510"
-          target="_blank"
-        >
-          <i class="pi pi-twitter text-2xl text-white"></i>
-        </a>
+<p-toolbar>
+  <div class="p-toolbar-group-start"></div>
+  <div class="p-toolbar-group-center"></div>
+  <div class="p-toolbar-group-end">
+    <div class="flex lg:px-20">
+      <div class="ml-auto ms-2 !leading-[initial] text-white">
+        <a class="mt-8">&copy; 510 {{ currentYear }}</a>
```

Comment:
`p-toolbar` has a purple bottom border, and I think we added that with the assumption that it would only be used in the header.

You can probably circumnavigate the issue by replacing `p-toolbar` with something like this (not tested):

```suggestion
<div class="text-end bg-navy px-2 py-2 lg:px-20 txt-body-m">
  &copy; 510 {{ currentYear }}
</div>
```